Dockerfile

```diff
@@ -33,9 +30,18 @@ COPY ./php.ini-development $PHP_INI_DIR/php.ini
 RUN rm -rf $OST_ROOT/*
 
 COPY . $OST_ROOT
+COPY start.sh /usr/local/bin/
 
 RUN rm -rf $OST_ROOT/setup
 
 RUN rm -f $OST_ROOT/php.ini-development
 
-RUN chmod 644 $OST_ROOT/include/ost-config.php
\ No newline at end of file
+RUN chmod 644 $OST_ROOT/include/ost-config.php
+
+ARG CRON_TASK="*/5 * * * * /usr/local/bin/php -c $PHP_INI_DIR/php.ini $OST_ROOT/api/cron.php"
+ARG CRON_FILE="${OST_ROOT}/crontab"
+RUN echo "${CRON_TASK}\n" >> $CRON_FILE
+RUN crontab -u www-data ${CRON_FILE}
+RUN rm ${CRON_FILE}
+
+CMD ["start.sh"]
```

Comment:
Does this work as a CMD or should it be and ENTRYPOINT?

Reply:
It works, but I'm not sure of the reason to use one over the other.  I was going off of the base image, which is at https://github.com/docker-library/php/blob/master/7.4/buster/apache/Dockerfile

Reply:
CMD is appended to the ENTRYPOINT basically. CMD can also be overridden where ENTRYPOINT can't. I think what runs when this image starts is `docker-php-entrypoint start.sh` now rather than `docker-php-entrypoint apache2-foreground`. If that works then lets ship it.

Reply:
I shelled into the running container and verified that both crond and apache were running.